### PR TITLE
Update intellij plugin to 1.1

### PR DESCRIPTION
**Pull request by rli**
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts [hunk 1]
```diff
@@ -73,22 +73,22 @@ afterEvaluate {
...
-        version = productProfile.sdkVersion
+        pluginName.set("aws-toolkit-jetbrains")
+        version.set(productProfile.sdkVersion)
+        plugins.set(productProfile.plugins.toList())
 
-        setPlugins(*productProfile.plugins)
-
-        downloadSources = flavor != IdeFlavor.IC
-        instrumentCode = flavor != IdeFlavor.RD
+        downloadSources.set(flavor != IdeFlavor.IC)
```
Comment (abrooksv) [inline buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts:80]: Hmm this should have had a && System.getenv("CI") != null

**Comment by abrooksv** [comment]
PR to fix the instrument issues: https://github.com/JetBrains/gradle-intellij-plugin/pull/668

**Comment by abrooksv** [comment]
Re-based but still blocked on 1.0.1

File: buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts [hunk 2]
```diff
@@ -65,92 +61,82 @@ tasks.processResources {
...
-    }
+tasks.withType<JavaExec> {
+    systemProperty("aws.toolkits.enableTelemetry", false)
+}
 
-    tasks.withType<Test>().all {
-        systemProperty("log.dir", "${Utils.stringInput(intellij.sandboxDirectory)}-test/logs")
-        systemProperty("testDataPath", project.rootDir.resolve("testdata").absolutePath)
+tasks.runIde {
+    val alternativeIde = System.getenv("ALTERNATIVE_IDE")
```
Comment (rli) [inline buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts:97]: another one

File: buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts [hunk 3]
```diff
@@ -62,92 +61,82 @@ tasks.processResources {
...
-    val flavor = toolkitIntelliJ.ideFlavor.get()
-    val productProfile = when (flavor) {
-        IdeFlavor.IC -> ideProfile.community
-        IdeFlavor.IU -> ideProfile.ultimate
-        IdeFlavor.RD -> ideProfile.rider
-    }
+intellij {
+    pluginName.set("aws-toolkit-jetbrains")
+    version.set(toolkitIntelliJ.productProfile().map { it.sdkVersion })
+    plugins.set(toolkitIntelliJ.productProfile().map { it.plugins.toMutableList() })
```
[the remark this answers is not in the document]
Comment (abrooksv) [reply buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts:67]: Sadly, yes

```
NONE_APPLICABLE] None of the following functions can be called with the arguments supplied.
set((Mutable)Iterable<Any!>?) defined in org.gradle.api.provider.ListProperty
set(Provider<out (Mutable)Iterable<Any!>!>!) defined in org.gradle.api.provider.ListProperty
```

File: buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts [hunk 4]
```diff
@@ -62,92 +61,82 @@ tasks.processResources {
...
 
-    intellij {
-        pluginName.set("aws-toolkit-jetbrains")
-        version.set(productProfile.sdkVersion)
-        plugins.set(productProfile.plugins.toList())
+    downloadSources.set(toolkitIntelliJ.ideFlavor.map { it == IdeFlavor.IC && System.getenv("CI") == null })
```
[the remark this answers is not in the document]
Comment (abrooksv) [reply buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts]: Fix them all including code artifact

File: intellij/build.gradle.kts [hunk 5]
```diff
@@ -24,10 +23,11 @@ val resharperDlls = configurations.create("resharperDlls") {
...
 
 intellij {
-    version = ideProfile.community.sdkVersion
-    pluginName = "aws-toolkit-jetbrains"
-    updateSinceUntilBuild = false
-    instrumentCode = false
+    version.set(ideProfile.community.sdkVersion)
+    pluginName.set("aws-toolkit-jetbrains")
+    updateSinceUntilBuild.set(false)
+
```
Comment (rli) [inline intellij/build.gradle.kts:29]: nit: extra new line here
```suggestion

```

**Review by rli — COMMENTED** [review]
LGTM: 
image

**Review by abrooksv — APPROVED** [review]

**Pull request merged**